memcache_proxy/dtog.go

```diff
@@ -33,6 +33,12 @@ import (
 	pb "google.golang.org/appengine/notreallyinternal/memcache"
 )
 
+const (
+	// Max App Engine Memcache value size (10^6 bytes).
+	// https://cloud.google.com/appengine/docs/python/memcache/
+	maxMemcacheValueSize = 1000000
```

Comment:
Note that although the Python API enforces 10^6 max (1 MB), the underlying system can handle 2^20 (1 MiB) of key+value+overhead.  This is exposed, for example, in the [Go API docs][1]: "maximum size of a cached data value is 1 MiB (2^20 bytes) minus the size of the key minus an implementation-dependent overhead, which is approximately 73 bytes"

[1]: https://cloud.google.com/appengine/docs/go/memcache/#Go_Limits